target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
It's not fine as it bloats all other targets and btrfs isn't really a great idea on nand either.